lib/Component.lua

```diff
@@ -121,6 +121,44 @@ function Component:render()
 	error(message, 0)
 end
 
+--[[
+	Tries to update the component's children, correctly handling the presence of
+	an error boundary.
+]]
+function Component:__updateChildren(reconciler, virtualNode)
```

Comment:
Not 100% sure, but is this function name a little misleading? The function's job is basically "render and handle errors", and the "render" part of it is, I think, not quite the same thing as updating children?

I don't really have any suggestions, though... just wanted to see what you thought.

Reply:
I picked it largely because when I initially wrote this it was an error handling wrapper around `reconciler.updateVirtualNodeChildren` - after I merged in the latest upstream changes that method went away, but the name for this didn't change. I've changed it to `__updateVirtualNode` to reflect it, but I don't have any super insightful reasoning for this name either!